internal/js-ast-utils/valueToNode.test.ts

```diff
@@ -0,0 +1,83 @@
+import {test} from "rome";
+import {valueToNode} from "./valueToNode";
+import {
+	jsArrayExpression,
+	jsBigIntLiteral,
+	jsBooleanLiteral,
+	jsNullLiteral,
+	jsNumericLiteral,
+	jsObjectExpression,
+	jsObjectProperty,
+	jsReferenceIdentifier,
+	jsStaticPropertyKey,
+	jsStringLiteral,
+} from "@internal/ast";
+import {createPropertyKey} from "./createPropertyKey";
+
+test(
+	"verify valueToNode returns the correct supported primitive type",
+	(t) => {
+		t.looksLike(valueToNode("string"), jsStringLiteral.quick("string"));
+		t.looksLike(valueToNode(true), jsBooleanLiteral.quick(true));
+		t.looksLike(valueToNode(123), jsNumericLiteral.quick(123));
+		t.looksLike(
+			valueToNode(BigInt(10)),
+			jsBigIntLiteral.create(jsStringLiteral.quick("10")),
```

Comment:
This is actually just using a quirk. `create` accepts an object with properties for the node. `JSBigIntLiteral` only has a single `value` property that's a `string`. Technically a `JSStringLiteral` would satisfy this type.

What you're looking for is the `quick` method that we have on nodes that only have a single property. This allows us to skip having an object entirely. (Side note: I don't like the quick name but couldn't think of anything better, open to suggestions there!)

```suggestion
			jsBigIntLiteral.quick("10"),
```